packager/media/base/stream_info.h

```diff
@@ -55,6 +55,8 @@ enum Codec {
   kCodecVorbis,
   kCodecMP3,
   kCodecAudioMaxPlusOne,
+  kCodecMha1,
+  kCodecMhm1,
```

Comment:
Can you move them before kCodecAudioMaxPlusOne, i.e. before line 57?

Reply:
Done.